.github/workflows/lint_test.yml

```diff
@@ -12,21 +12,21 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.16.x]
+        go-version: [1.17.x]
```

Comment:
had to update per https://golangshowcase.com/question/github-action-for-golangci-lint-fails-with-cant-load-fmt